### ISSUE TITLE
Receiving notifications on issues opened from Slack

### ISSUE THREAD

**Issue by MynahMarie** [lotus-coding-school/master-reference#13]
When we send direct messages from Slack and we would like the person we are messaging to follow issues posted on a specific repo, we can add the github app to the conversation by adding an app and choosing github. Once this is done, we can subscribe a repo to our conversation by typing the command `/github subscribe <organization>/<repo>`. To open a new issue, the command is `/github open <organization/<repo>`. @shiryz hope this helps! :smiley_cat:
